packages/discord.js/typings/index.d.ts

```diff
@@ -4999,20 +4999,20 @@ export interface Caches {
   // TODO: GuildManager: [manager: typeof GuildManager, holds: typeof Guild];
   GuildMemberManager: [manager: typeof GuildMemberManager, holds: typeof GuildMember];
   GuildBanManager: [manager: typeof GuildBanManager, holds: typeof GuildBan];
-  GuildForumThreadManager: [manager: typeof GuildForumThreadManager, holds: typeof ThreadChannel<true>];
+  GuildForumThreadManager: [manager: typeof GuildForumThreadManager, holds: AnyThreadChannel<true>];
```

Comment:
Pretty sure this still needs typeof

Reply:
Nope! `ThreadChannel` is an actual class that exists at runtime, that's why we need `typeof`. `AnyThreadChannel` is a type alias for different types of threads.

Reply:
Can you add a test please that this doesn't break anything 🙏